docs/build_mozc_in_windows.md

```diff
@@ -160,6 +166,12 @@ Note that you can specify `--qtdir=` option instead of `--noqt` in GYP phase sin
 
 ## Build with Bazel (experimental)
 
+If you installed Bazel via [Scoop](https://scoop.sh), it is recommended to install MSYS2 via Scoop, too.
```

Comment:
Let's make a section of `Tips for Bazel setup` at the end of this section, and move this line there.